doc/development_process/dev_env_and_tools.rst

```diff
@@ -268,13 +268,21 @@ To be discussed in a meeting
 ============================
 
 =============  ===============================================================
-Labels         ``API``, ``dev-review``, ``TSC``
-Applicable to  PRs  and issues
+Labels         ``meeting: {API|dev-review}``
```

Comment:
What about TSC?

Reply:
I remove it from this set, and left it to be just a review type & time label (group below), see comment above:
https://github.com/zephyrproject-rtos/zephyr/pull/15756#issuecomment-489037033